modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/company_mail.jsp

```diff
@@ -7,12 +7,19 @@
 
 <%@ include file="/init.jsp" %>
 
+<%
+long companyId = company.getCompanyId();
+
+Function<String, String> defaultValueFunction = key -> StringPool.BLANK;
+%>
+
 <portlet:actionURL name="/server_admin/edit_server" var="actionURL" />
 
 <portlet:renderURL var="redirectURL" />
 
 <aui:form action="<%= actionURL %>" cssClass="sheet-lg" method="post" name="fm">
 	<aui:input name="<%= Constants.CMD %>" type="hidden" value="updateMail" />
+	<aui:input name="preferencesCompanyId" type="hidden" value="<%= String.valueOf(companyId) %>" />
```

Comment:
@marianoalvarosaiz This is not need.

It is not that easy to find this is not necessary as you have to search the variable in your new mail_fields.jsp.

This kind of issue can be seen easily by modifying mail_fileds.jsp directly rather than copying the logic to mail.jsp and company_mail.jsp, then extract. 

By modifying mail_fileds.jsp directly, it can also show me easily what you changed and what you did not change. While, with your current commit history, it is really hard to see and I have to compare the new mail_fields.jsp and mail_fields.jspf manually.

Please try to rearrange your commit so that it would be easier for us to track what's changed to avoid any potential mistakes we can mde.

Thanks.
Tina